iris-mpc-cpu/proto/party_node.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package party_node;
+
+service PartyNode {
+    rpc SendMessage (SendRequest) returns (SendResponse);
+}
```

Comment:
Some things to think about here:
1) Should we use unidirectional streaming gRPC for this, instead of request/response gRPC?
 (i.e., (`rpc SendMessage (stream SendRequest) returns (SendResponse);`) which can then be drained directly into the async_channel?) Probably a bit more complex with timeouts etc, but should hopefully be more performant.
2) should the message just be the data, and the sender_id and session_id be metadata that is sent via, headers etc? This would then also help a bit in the streaming case, since it is not part of every request?

Reply:
Done with the latter. Regarding the former, I can give it a try in the next PR.

Reply:
Yeah I think without the streaming it does not make too much sense to do it since it is just more annoying to grab the fields from the metadata, the only real advantage would be to have the metadata only send once for the streaming call instead of for each message

Reply:
Streaming seems to work, so I'll leave this metadata change here.